tests/testthat/test_pipeop_targettrafosimple.R

```diff
@@ -0,0 +1,74 @@
+context("PipeOpTargetTrafoSimple")
```

Comment:
So the following currently breaks, as it would require a `PredictionClassif` -> `PredictionRegr` conversion
- Task: `boston_housing`, `trafo = x > 20`,  inverter if (x) 19 else 21 (then we'd need a classif learner)

While the following might work. This is actually somewhat relevant.
- Task: `boston_housing`,`trafo = x > 20`,  inverter `identity`? This should probably be possible.

Reply:
Could we test that and possibly add a fixme that we might somehow enable the first option in the future?

Reply:
I added two more tests here, one that goes from  `PredictionClassif` -> `PredictionRegr` and one the other way. Both are not the most meaningful ones but it is possible. For most cases to actually be meaningful the user will rely on setting the `$state` and rely on this for the inversion (like statet by you below).